tests/integration_test/opencensus_tracing_test/opencensus_tracing_test.go

```diff
@@ -612,6 +612,92 @@ func TestTraceContextPropagationHeaders(t *testing.T) {
 	}
 }
 
+func TestTraceContextPropagationHeadersForScCheck(t *testing.T) {
+	t.Parallel()
+
+	traceId := "0af7651916cd43dd8448eb211c80319c"
+	spanId := "b7ad6b7169203331"
+	incomingTraceContexts := map[string][]string{
+		"traceparent": {
+			createTraceparentContext(traceId, spanId),
+		},
+	}
+
+	tests := []struct {
+		desc                 string
+		tracingSampleRate    float32
+		expectedScReqHeaders map[string][]string
+	}{
+		{
+			desc:              "SC Check receives trace context propagation header.",
+			tracingSampleRate: 1,
+			expectedScReqHeaders: map[string][]string{
+				// Only the trace id is checked. Span id should be changed.
+				"Traceparent": {
+					createTraceparentContextPrefix(traceId),
+				},
```

Comment:
what is the different between this case and case 3?

should we also expect "X-Cloud-Trace-Context" in this case?

Reply:
Fixed. Merged the 2 cases together.